content/en/account_management/rbac/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Role Based Access Control
 kind: documentation
+disable_toc: true
```

Comment:
Disabling the TOCs is not necessary. This only affects the page. To disable search add `private: true`